src/bar_ros/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>bar_ros</name>
+  <version>0.0.1</version>
+  <description>The bar package</description>
+
+  <maintainer email="rre@robopec.com">rre</maintainer>
+  <license>BSD</license>
+
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
```

Comment:
is this 'condition' attribute currently supported?

Reply:
Yes. It's part of [REP-149: Package Manifest Format Three Specification](https://www.ros.org/reps/rep-0149.html).